pkg/util/selinux/selinux_linux.go

```diff
@@ -50,3 +50,8 @@ func (_ *realSELinuxRunner) Getfilecon(path string) (string, error) {
 	}
 	return selinux.FileLabel(path)
 }
+
+// FileLabel returns the SELinux label for this path or returns an error.
```

Comment:
does this comment belong here? We call ```SetFileLabel``` here not ```FileLabel```; and the former doesn't return label or error, just error.